### PR TITLE
🐛 Source Pipedrive: Normalization Error Person Table: Invalid timestamp

### LINKED ISSUE #6406: Pipedrive Normalization Error Person Table: Invalid timestamp
## Enviroment
- **Airbyte version**:  0.29-21
- **OS Version / Instance**: Ubuntu 20.04
- **Deployment**: Docker 19.03.12
- **Source Connector and version**:  Pipedrive 0.1.3
- **Destination Connector and version**:  BigQuery 0.4.0
- **Severity**: High
- **Step where error happened**: Sync job 

## Current Behavior
Person table is not synched to BigQuery and normalisation fails with error because of invalid timestamp.

## Expected Behavior
Normalisation should be processed and data should be synched to table in BigQuery.

## Logs

<details>
<summary>LOG</summary> 

```

2021-09-23 19:06:42 INFO () WorkerRun(call):62 - Executing worker wrapper. Airbyte version: 0.29.21-alpha
2021-09-23 19:06:43 INFO () TemporalAttemptExecution(get):114 - Executing worker wrapper. Airbyte version: 0.29.21-alpha
2021-09-23 19:06:43 WARN () Databases(createPostgresDatabaseWithRetry):58 - Waiting for database to become available...
2021-09-23 19:06:43 INFO () JobsDatabaseInstance(lambda$static$2):45 - Testing if jobs database is ready...
2021-09-23 19:06:44 INFO () Databases(createPostgresDatabaseWithRetry):75 - Database available!
2021-09-23 19:06:44 WARN () JsonMetaSchema(newValidator):338 - Unknown keyword example - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2021-09-23 19:06:44 INFO () DefaultReplicationWorker(run):102 - start sync worker. job id: 18 attempt id: 0
2021-09-23 19:06:44 INFO () DefaultReplicationWorker(run):111 - configured sync modes: {null.persons=full_refresh - append}
2021-09-23 19:06:44 INFO () DefaultAirbyteDestination(start):78 - Running destination...
2021-09-23 19:06:44 INFO () LineGobbler(voidCall):85 - Checking if airbyte/destination-bigquery:0.4.0 exists...
2021-09-23 19:06:44 INFO () LineGobbler(voidCall):85 - airbyte/destination-bigquery:0.4.0 was found locally.
2021-09-23 19:06:44 INFO () DockerProcessFactory(create):146 - Preparing command: docker run --rm --init -i -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -w /data/18/0 --network host --log-driver none airbyte/destination-bigquery:0.4.0 write --config destination_config.json --catalog destination_catalog.json
2021-09-23 19:06:44 INFO () LineGobbler(voidCall):85 - Checking if airbyte/source-pipedrive:0.1.3 exists...
2021-09-23 19:06:44 INFO () LineGobbler(voidCall):85 - airbyte/source-pipedrive:0.1.3 was found locally.
2021-09-23 19:06:44 INFO () DockerProcessFactory(create):146 - Preparing command: docker run --rm --init -i -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -w /data/18/0 --network host --log-driver none airbyte/source-pipedrive:0.1.3 read --config source_config.json --catalog source_catalog.json
2021-09-23 19:06:44 INFO () DefaultReplicationWorker(lambda$getDestinationOutputRunnable$3):246 - Destination output thread started.
2021-09-23 19:06:44 INFO () DefaultReplicationWorker(run):139 - Waiting for source thread to join.
2021-09-23 19:06:44 INFO () DefaultReplicationWorker(lambda$getReplicationRunnable$2):210 - Replication thread started.
2021-09-23 19:06:45 INFO () DefaultAirbyteStreamFactory(internalLog):110 - Starting syncing SourcePipedrive
2021-09-23 19:06:45 INFO () DefaultAirbyteStreamFactory(internalLog):110 - Syncing stream: persons 
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.d.b.BigQueryDestination(main):356 - {} - starting destination: class io.airbyte.integrations.destination.bigquery.BigQueryDestination
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.b.IntegrationRunner(run):96 - {} - Running integration: io.airbyte.integrations.destination.bigquery.BigQueryDestination
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.b.IntegrationCliParser(parseOptions):135 - {} - integration args: {catalog=destination_catalog.json, write=null, config=destination_config.json}
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.b.IntegrationRunner(run):100 - {} - Command: WRITE
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.b.IntegrationRunner(run):101 - {} - Integration config: IntegrationConfig{command=WRITE, configPath='destination_config.json', catalogPath='destination_catalog.json', statePath='null'}
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - {} - Unknown keyword examples - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [33mWARN[m c.n.s.JsonMetaSchema(newValidator):338 - {} - Unknown keyword airbyte_secret - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.d.b.BigQueryDestination(getLoadingMethod):331 - {} - Selected loading method is set to: STANDARD
2021-09-23 19:06:47 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:47 [32mINFO[m i.a.i.d.b.BigQueryDestination(isKeepFilesInGcs):344 - {} - All tmp files will be removed from GCS when migration is finished
2021-09-23 19:06:49 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:49 [32mINFO[m i.a.i.d.b.BigQueryUtils(createTable):112 - {} - Table: GenericData{classInfo=[datasetId, projectId, tableId], {datasetId=pipedrive, tableId=_airbyte_tmp_cpj_pipedrive_persons}} created successfully
2021-09-23 19:06:51 INFO () DefaultAirbyteStreamFactory(internalLog):110 - Read 761 records from persons stream
2021-09-23 19:06:51 INFO () DefaultAirbyteStreamFactory(internalLog):110 - Finished syncing SourcePipedrive
2021-09-23 19:06:51 INFO () DefaultReplicationWorker(run):141 - Source thread complete.
2021-09-23 19:06:51 INFO () DefaultReplicationWorker(run):142 - Waiting for destination thread to join.
2021-09-23 19:06:52 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:52 [32mINFO[m i.a.i.b.FailureTrackingAirbyteMessageConsumer(close):80 - {} - Airbyte message consumer: succeeded.
2021-09-23 19:06:52 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:52 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(close):163 - {} - Started closing all connections
2021-09-23 19:06:53 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:53 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):278 - {} - Waiting for jobs to be finished/closed
2021-09-23 19:06:55 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:55 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):295 - {} - Migration finished with no explicit errors. Copying data from tmp tables to permanent
2021-09-23 19:06:56 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:56 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(copyTable):359 - {} - successfully copied tmp table: GenericData{classInfo=[datasetId, projectId, tableId], {datasetId=pipedrive, tableId=_airbyte_tmp_cpj_pipedrive_persons}} to final table: GenericData{classInfo=[datasetId, projectId, tableId], {datasetId=pipedrive, tableId=_airbyte_raw_pipedrive_persons}}
2021-09-23 19:06:56 INFO () JsonSchemaValidator(test):76 - JSON schema validation failed. 
errors: $: null found, object expected
2021-09-23 19:06:56 ERROR () DefaultAirbyteStreamFactory(lambda$create$1):83 - Validation failed: null
2021-09-23 19:06:56 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:56 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):307 - {} - Removing tmp tables...
2021-09-23 19:06:56 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:56 [32mINFO[m i.a.i.d.b.BigQueryRecordConsumer(closeNormalBigqueryStreams):309 - {} - Finishing destination process...completed
2021-09-23 19:06:56 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:56 [32mINFO[m i.a.i.b.IntegrationRunner(run):153 - {} - Completed integration: io.airbyte.integrations.destination.bigquery.BigQueryDestination
2021-09-23 19:06:56 INFO () DefaultAirbyteStreamFactory(lambda$create$0):73 - 2021-09-23 19:06:56 [32mINFO[m i.a.i.d.b.BigQueryDestination(main):358 - {} - completed destination: class io.airbyte.integrations.destination.bigquery.BigQueryDestination
2021-09-23 19:06:56 INFO () DefaultReplicationWorker(run):144 - Destination thread complete.
2021-09-23 19:06:56 INFO () DefaultReplicationWorker(run):172 - sync summary: io.airbyte.config.ReplicationAttemptSummary@297fad54[status=completed,recordsSynced=761,bytesSynced=1300241,startTime=1632424004303,endTime=1632424016786]
2021-09-23 19:06:56 INFO () DefaultReplicationWorker(run):181 - Source did not output any state messages
2021-09-23 19:06:56 WARN () DefaultReplicationWorker(run):192 - State capture: No state retained.
2021-09-23 19:06:56 INFO () TemporalAttemptExecution(get):135 - Stopping cancellation check scheduling...
2021-09-23 19:06:56 INFO () SyncWorkflow$ReplicationActivityImpl(replicate):184 - sync summary: io.airbyte.config.StandardSyncOutput@6242b963[standardSyncSummary=io.airbyte.config.StandardSyncSummary@1f6fd0eb[status=completed,recordsSynced=761,bytesSynced=1300241,startTime=1632424004303,endTime=1632424016786],state=<null>,outputCatalog=io.airbyte.protocol.models.ConfiguredAirbyteCatalog@14045171[streams=[io.airbyte.protocol.models.ConfiguredAirbyteStream@106420ce[stream=io.airbyte.protocol.models.AirbyteStream@679a0a00[name=pipedrive_persons,jsonSchema={"type":"object","$schema":"http://json-schema.org/draft-07/schema#","properties":{"id":{"type":["null","integer"]},"name":{"type":["null","string"]},"email":{"type":["null","array"]},"label":{"type":["null","integer"]},"phone":{"type":["null","array"]},"org_id":{"type":["null","integer"]},"add_time":{"type":["null","string"],"format":"date-time"},"cc_email":{"type":["null","string"]},"org_name":{"type":["null","string"]},"owner_id":{"type":["null","integer"]},"last_name":{"type":["null","string"]},"company_id":{"type":["null","integer"]},"first_char":{"type":["null","string"]},"first_name":{"type":["null","string"]},"owner_name":{"type":["null","string"]},"picture_id":{"type":["null","object"],"properties":{"value":{"type":["null","integer"]},"item_id":{"type":["null","integer"]},"add_time":{"type":["null","string"],"format":"date-time"},"pictures":{"type":["null","object"],"properties":{"128":{"type":["null","string"]},"512":{"type":["null","string"]}}},"item_type":{"type":["null","string"]},"active_flag":{"type":["null","boolean"]},"update_time":{"type":["null","string"]},"added_by_user_id":{"type":["null","integer"]}}},"visible_to":{"type":["null","string"]},"active_flag":{"type":["null","boolean"]},"files_count":{"type":["null","integer"]},"notes_count":{"type":["null","integer"]},"update_time":{"type":["null","string"],"format":"date-time"},"followers_count":{"type":["null","integer"]},"won_deals_count":{"type":["null","integer"]},"activities_count":{"type":["null","integer"]},"last_activity_id":{"type":["null","integer"]},"lost_deals_count":{"type":["null","integer"]},"next_activity_id":{"type":["null","integer"]},"open_deals_count":{"type":["null","integer"]},"closed_deals_count":{"type":["null","integer"]},"last_activity_date":{"type":["null","string"],"format":"date-time"},"next_activity_date":{"type":["null","string"],"format":"date-time"},"next_activity_time":{"type":["null","string"],"format":"date-time"},"email_messages_count":{"type":["null","integer"]},"done_activities_count":{"type":["null","integer"]},"last_incoming_mail_time":{"type":["null","string"],"format":"date-time"},"last_outgoing_mail_time":{"type":["null","string"],"format":"date-time"},"related_won_deals_count":{"type":["null","integer"]},"undone_activities_count":{"type":["null","integer"]},"related_lost_deals_count":{"type":["null","integer"]},"related_open_deals_count":{"type":["null","integer"]},"related_closed_deals_count":{"type":["null","integer"]},"participant_open_deals_count":{"type":["null","integer"]},"participant_closed_deals_count":{"type":["null","integer"]}}},supportedSyncModes=[full_refresh, incremental],sourceDefinedCursor=true,defaultCursorField=[update_time],sourceDefinedPrimaryKey=[[id]],namespace=<null>,additionalProperties={}],syncMode=full_refresh,cursorField=[update_time],destinationSyncMode=append,primaryKey=[[id]],additionalProperties={}]],additionalProperties={}]]
2021-09-23 19:06:56 INFO () TemporalAttemptExecution(get):114 - Executing worker wrapper. Airbyte version: 0.29.21-alpha
2021-09-23 19:06:56 WARN () Databases(createPostgresDatabaseWithRetry):58 - Waiting for database to become available...
2021-09-23 19:06:56 INFO () JobsDatabaseInstance(lambda$static$2):45 - Testing if jobs database is ready...
2021-09-23 19:06:56 INFO () Databases(createPostgresDatabaseWithRetry):75 - Database available!
2021-09-23 19:06:56 INFO () DefaultNormalizationWorker(run):63 - Running normalization.
2021-09-23 19:06:56 INFO () LineGobbler(voidCall):85 - Checking if airbyte/normalization:0.1.44 exists...
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 - airbyte/normalization:0.1.44 was found locally.
2021-09-23 19:06:57 INFO () DockerProcessFactory(create):146 - Preparing command: docker run --rm --init -i -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -w /data/18/0/normalize --network host --log-driver none airbyte/normalization:0.1.44 run --integration-type bigquery --config destination_config.json --catalog destination_catalog.json
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 - Running: transform-config --config destination_config.json --integration-type bigquery --out /data/18/0/normalize
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 - Namespace(config='destination_config.json', integration_type=<DestinationType.bigquery: 'bigquery'>, out='/data/18/0/normalize')
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 - transform_bigquery
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 - Running: transform-catalog --integration-type bigquery --profile-config-dir /data/18/0/normalize --catalog destination_catalog.json --out /data/18/0/normalize/models/generated/ --json-column _airbyte_data
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 - Processing destination_catalog.json...
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_ab1.sql from pipedrive_persons
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_ab2.sql from pipedrive_persons
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_ab3.sql from pipedrive_persons
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_tables/pipedrive/pipedrive_persons.sql from pipedrive_persons
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_picture_id_ab1.sql from pipedrive_persons/picture_id
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_picture_id_ab2.sql from pipedrive_persons/picture_id
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_picture_id_ab3.sql from pipedrive_persons/picture_id
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_tables/pipedrive/pipedrive_persons_picture_id.sql from pipedrive_persons/picture_id
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_picture_id_pictures_ab1.sql from pipedrive_persons/picture_id/pictures
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_picture_id_pictures_ab2.sql from pipedrive_persons/picture_id/pictures
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_ctes/pipedrive/pipedrive_persons_picture_id_pictures_ab3.sql from pipedrive_persons/picture_id/pictures
2021-09-23 19:06:57 INFO () LineGobbler(voidCall):85 -   Generating airbyte_tables/pipedrive/pipedrive_persons_picture_id_pictures.sql from pipedrive_persons/picture_id/pictures
2021-09-23 19:06:57 ERROR () LineGobbler(voidCall):85 - cat: /data/18/0/normalize/localsshport.json: No such file or directory
2021-09-23 19:06:58 INFO () LineGobbler(voidCall):85 - Running with dbt=0.19.0
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - There are 1 unused configuration paths:
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - - models.airbyte_utils.generated.airbyte_views
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - 
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - Found 12 models, 0 tests, 0 snapshots, 0 analyses, 423 macros, 0 operations, 0 seed files, 1 source, 0 exposures
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - 
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - 19:07:01 | Concurrency: 32 threads (target='prod')
2021-09-23 19:07:01 INFO () LineGobbler(voidCall):85 - 19:07:01 | 
2021-09-23 19:07:02 INFO () LineGobbler(voidCall):85 - 19:07:02 | 1 of 3 START table model pipedrive.pipedrive_persons......................................................... [RUN]
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 19:07:06 | 1 of 3 ERROR creating table model pipedrive.pipedrive_persons................................................ [[31mERROR[0m in 4.50s]
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 19:07:06 | 2 of 3 SKIP relation pipedrive.pipedrive_persons_picture_id.................................................. [[33mSKIP[0m]
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 19:07:06 | 3 of 3 SKIP relation pipedrive.pipedrive_persons_picture_id_pictures......................................... [[33mSKIP[0m]
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 19:07:06 | 
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 19:07:06 | Finished running 3 table models in 5.45s.
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - [31mCompleted with 1 error and 0 warnings:[0m
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - [33mDatabase Error in model pipedrive_persons (models/generated/airbyte_tables/pipedrive/pipedrive_persons.sql)[0m
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 -   Invalid timestamp: '08:30:00'
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 -   compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/pipedrive/pipedrive_persons.sql
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - 
2021-09-23 19:07:06 INFO () LineGobbler(voidCall):85 - Done. PASS=0 WARN=0 ERROR=1 SKIP=2 TOTAL=3
2021-09-23 19:07:07 INFO () TemporalAttemptExecution(lambda$getWorkerThread$1):168 - Completing future exceptionally...
io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:77) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:38) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:165) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:74) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	... 3 more
	Suppressed: io.airbyte.workers.WorkerException: Normalization process wasn't successful
		at io.airbyte.workers.normalization.DefaultNormalizationRunner.close(DefaultNormalizationRunner.java:151) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:62) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:38) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:165) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at java.lang.Thread.run(Thread.java:832) [?:?]
2021-09-23 19:07:07 INFO () TemporalAttemptExecution(get):135 - Stopping cancellation check scheduling...
2021-09-23 19:07:07 WARN () POJOActivityTaskHandler$POJOActivityImplementation(execute):243 - Activity failure. ActivityId=9f66c9e9-28a0-308b-b604-d17a9cef348d, activityType=Normalize, attempt=1
java.util.concurrent.ExecutionException: io.airbyte.workers.WorkerException: Normalization Failed.
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2063) ~[?:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.get(TemporalAttemptExecution.java:133) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.temporal.SyncWorkflow$NormalizationActivityImpl.normalize(SyncWorkflow.java:291) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:564) ~[?:?]
	at io.temporal.internal.sync.POJOActivityTaskHandler$POJOActivityInboundCallsInterceptor.execute(POJOActivityTaskHandler.java:277) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.sync.POJOActivityTaskHandler$POJOActivityImplementation.execute(POJOActivityTaskHandler.java:216) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.sync.POJOActivityTaskHandler.handle(POJOActivityTaskHandler.java:181) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:192) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.worker.ActivityWorker$TaskHandlerImpl.handle(ActivityWorker.java:154) ~[temporal-sdk-1.0.4.jar:?]
	at io.temporal.internal.worker.PollTaskExecutor.lambda$process$0(PollTaskExecutor.java:73) ~[temporal-sdk-1.0.4.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) [?:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:77) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:38) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:165) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	... 1 more
Caused by: io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:74) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:38) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:165) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
	... 1 more
	Suppressed: io.airbyte.workers.WorkerException: Normalization process wasn't successful
		at io.airbyte.workers.normalization.DefaultNormalizationRunner.close(DefaultNormalizationRunner.java:151) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:62) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:38) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$1(TemporalAttemptExecution.java:165) ~[io.airbyte-airbyte-workers-0.29.21-alpha.jar:?]
		at java.lang.Thread.run(Thread.java:832) [?:?]

```

</details>

## Steps to Reproduce
1. Setup Pipedrive to BigQuery Connector
2. Export Person Table from Pipedrive with default normalization
3. Start Sync job

## Are you willing to submit a PR?
No

### PR DESCRIPTION
**Pull request by gaart**
## What
Fixed bugs with incorrect field types of some streams

## How

- Update stream schemas according to the API documentation. Change the format of some fields from date-time to actual date or time
- Update primary key of ActivityFields thread

#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated 
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [x] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [x] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [x] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

### CODE REVIEW

File: airbyte-integrations/connectors/source-pipedrive/source_pipedrive/schemas/deals.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     },
     "last_activity_date": {
       "type": ["null", "string"],
-      "format": "date-time"
+      "format": "time"
```
Comment (vitaliizazmic) [inline airbyte-integrations/connectors/source-pipedrive/source_pipedrive/schemas/deals.json:71]: Are you sure that last activity date has time format?

**Review by vitaliizazmic — COMMENTED** [review]
LGTM, only small question.

**Review by keu — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

**Comment by martin-restack** [comment]
The issue isn't resolved yet unfortunately. 
https://github.com/airbytehq/airbyte/issues/64
[logs-28-0.txt](https://github.com/airbytehq/airbyte/files/7336845/logs-28-0.txt)
06

**Pull request merged**